### DISCUSSION TITLE
How to center notes on tables figures etc.

### DISCUSSION THREAD

**Discussion by friosavila** [quarto-dev/quarto-cli#8705] [Q&A] [answered]
### Description

Good morning. 
Im currently using Quarto 1.4, and trying to better understand the new cross-reference features.
For that purpose i create different tables using different formats (figure, MD, HTML and Latex) with the following code

````

::: {#tbl-table1  tbl-pos="H"}
![](https://imgflip.com/s/meme/Table-Flip-Guy.jpg){fig-align=center}

Note: This was a png figure

Table made with a picture
:::


::: {#tbl-table2}

| Header 1 | Header 2 | Header 3 |
| -------- | -------- | -------- |
| Cell 1   | Cell 2   | Cell 3   |
| Cell 4   | Cell 5   | Cell 6   |

Note: Random note

Table using Markdown code
:::


::: {#tbl-table3 tbl-pos="H"}

```{=html}
<table>
  <tr><th>Header 1</th><th>Header 2</th><th>Header 3</th></tr>
  <tr><td>Cell 1</td><td>Cell 2</td><td>Cell 3</td></tr>
  <tr><td>Cell 4</td><td>Cell 5</td><td>Cell 6</td></tr>
</table>
```

Note: Random note

Table using HTML code
:::

::: {#tbl-table4 tbl-pos="H"}
\begin{tabular}{|c|c|c|}
\hline
Header 1 & Header 2 & Header 3 \\
\hline
Cell 1 & Cell 2 & Cell 3 \\
Cell 4 & Cell 5 & Cell 6 \\
\hline
\end{tabular}

Note: Random note

Table using Latex code
:::
````

My question relates to cases 2 and 3. (MD and HTML). Is there a way to center the "note", so it looks like case 1 and 4?

[Untitled-2.pdf](https://github.com/quarto-dev/quarto-cli/files/14241572/Untitled-2.pdf)

Thank you.

**Comment by cscheid** [comment, marked as answer]
All your use cases have multiple paragraphs of content in a div. That's not syntax we have considered deeply; I would suggest first trying to wrap the "content" of your float with a div by itself. I haven't tested the following, but something like:

````
::: {#tbl-table2}

::: {}

| Header 1 | Header 2 | Header 3 |
| -------- | -------- | -------- |
| Cell 1   | Cell 2   | Cell 3   |
| Cell 4   | Cell 5   | Cell 6   |

Note: Random note

:::

Table using Markdown code
:::
````

In the context of LaTeX specifically: You might need to, for now, use a hack like adding latex raw blocks that with `\centering{` and `}` at the beginning and end of the div.

The general principle is that we attempt interpret the first block of the div as the "content", and the second block as the caption. Our code might accidentally do the right thing right now with more than two blocks, but I suspect it's a less-tested path in general.

**Comment by friosavila** [comment]
Thank you!
I just thought there could be an option I was missing. 
In any case, the current behavior is enough for my needs. But was trying to consider other scenarios.
Best wishes
